src/main/scala/org/broadinstitute/dsde/firecloud/service/FireCloudRequestBuilding.scala

```diff
@@ -41,6 +44,8 @@ trait FireCloudRequestBuilding extends spray.httpx.RequestBuilding {
   }
 
   // with great power comes great responsibility!
-  def addAdminCredentials = addCredentials(OAuth2BearerToken(HttpGoogleServicesDAO.getAdminUserAccessToken))
+  def addAdminCredentials = addCredentials(OAuth2BearerToken(HttpGoogleServicesDAO.getAdminUserAccessToken)) ~> addFireCloudCredentials
```

Comment:
I believe that addAdminCredentials is used twice in the orch codebase, both in ProfileClient.

First usage talks to Rawls. This usage needs the admin creds, but does NOT need the Thurloe shared secret.
Second usage talks to Thurloe. This usage needs the shared secret, but NOT the admin creds.

I suggest leaving addAdminCredentials as-is (i.e. do not include addFireCloudCredentials), and then change the second usage - the one that talks to Thurloe - to remove addAdminCredentials and replace it with addFireCloudCredentials.